docs/pull_request_template.md

```diff
@@ -0,0 +1,38 @@
+### All Submissions:
+
+* [ ] Have you followed the guidelines in our **CONTRIBUTING** document?
+* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
+
+<!-- You can erase any parts of this template not applicable to your Pull Request. -->
+
+
+<!-- These have been added according to the rOpenSci guidelines. -->
+<!-- From: https://www.talater.com/open-source-templates/#/page/99 -->
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue. List the bug ID if applicable)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing core functionality to change) 
+
+
+### New Feature Submissions checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
```

Comment:
I think we need to add something like: Does you new feature comply with existing model APIs? e.g. for models, optimisation algorithms, objective functions
We then need to link to a new document describing each of those APIs...

Reply:
Or if we have API invariance tests implemented, it could be about ensuring that it's been added to the common test suite.

Reply:
Just using the wording I've suggested I think we can merge this PR before having the detailed documents + API invariance tests. I've opened issue #51 accordingly

Reply:
Addressed by https://github.com/josephguillaume/hydromad/pull/50/commits/0f6ae2630db514adfb2d772344ff90d660e1f48a